spec/extend-tests/222_test_newline_near_combinator.hrx

```diff
@@ -15,3 +15,8 @@
 .b y {
   a: b;
 }
+
+<===> output-libsass.css
```

Comment:
Since this test is marked as ignored for Dart Sass, it should probably only have a single `output.css` file, rather than a LibSass-specific one.